src/keymap.cpp

```diff
@@ -589,9 +595,13 @@ void KeyMap::handle_action(const std::string& action,
 			throw ConfigHandlerException(
 				ActionHandlerStatus::TOO_FEW_PARAMS);
 		std::string context = "all";
-		if (params.size() >= 2)
-			context = params[1];
-		unset_key(params[0], context);
+		if (params[0] == "-a") {
+			unset_all_keys();
+		} else {
+			if (params.size() >= 2)
```

Comment:
Since you've touched that code now, can you also add curly braces around the block?